cmd/util/confighelpers/configuration.go

```diff
@@ -305,7 +306,7 @@ func DumpConfig(k *koanf.Koanf, extraOverrideFields map[string]interface{}) erro
 		return fmt.Errorf("error removing extra parameters before dump: %w", err)
 	}
 
-	c, err := k.Marshal(koanfjson.Parser())
+	c, err := k.Marshal(json.Parser())
```

Comment:
why?

Reply:
Oh. I just noticed that the same package "json" package was imported both as "json" and "koanfjson" and decided to remove the duplicate import. I suppose I could have renamed the other uses in the file to "koanfjson" and removed the more idiomatic import.

But, also, I guess that change isn't really related to the main purpose of this PR. I was just "tidying up" while I was in this file. Should I revert that part and do the cleanup in a different PR?